pudb/remote.py

```diff
@@ -139,8 +141,10 @@ def __init__(
 
         if term_size is None:
             try:
-                s = os.get_terminal_size()
-                term_size = (s.columns, s.lines)
+                term_size = tuple(map(int, PUDB_TERM_SIZE.split("x")))
+                if len(term_size) != 2:
```

Comment:
Didn't spot this yesterday: I don't like that this is silently ignored if the format isn't right. It'd be better to raise. (Also above.)

Reply:
Done. Now if the `PDB_TERM_SIZE` value is invalid, one of the following exceptions will be raised:

- `ValueError: invalid literal for int() with base 10`
- `ValueError: PUDB_TERM_SIZE should have two dimensions`